app/src/utils/notifications.js

```diff
@@ -0,0 +1,30 @@
+import PushNotification from 'react-native-push-notification';
+
+PushNotification.configure({
+  onNotification: function(notification) {
```

Comment:
If all this is gonna do is log this information, we can safely make do without it.
What do you think?

Reply:
Yeah true.